docs/manual/common/gettingstarted/IDEs.md

```diff
@@ -84,9 +88,9 @@ Now update all projects declarations in your Lagom build to use the newly create
 
 Finally, you will need to regenerate the Eclipse project files to account for the changes made in the build file. Go back to the sbt console, type `reload` to reload the build file, and type `eclipse`. Once the `eclipse` task completes, go back to Eclipse, hit F5 to refresh all previously imported projects, and you are done.
 
-### IntelliJ
+### <a name="sbt-intellij"></a>Importing sbt projects into IntelliJ
 
-Because the Lagom build file is written in sbt, we recommend to install the IntelliJ IDEA sbt plugin to facilitate importing projects. Unfortunately, for sbt plugin to properly work, you will need to install the Scala plugin as well. To do so, open the `Plugins` modal for installing plugins, search for the "SBT" plugin. If no match is found, you'll have to hit "Browse".
+Because the Lagom build file is written in sbt, we recommend that you install the IntelliJ IDEA sbt plugin to facilitate importing projects. The plugin has a dependency on the Scala plugin, so you will need to install it first. To do so, open the `Plugins` modal for installing plugins, search for the "SBT" plugin. If no match is found, you'll have to hit "Browse".
```

Comment:
I think these instructions were written before the introduction of Maven support. I would take out "Because the Lagom build file is written in sbt" now, as it could be confusing and isn't really necessary.